### PR TITLE
Update list-of-terms.md

### PR DESCRIPTION
**Pull request by hannaparks**

### CODE REVIEW

File: company/about-mattermost/list-of-terms.md [hunk 1]
```diff
@@ -24,6 +22,14 @@ Refers to both an [As Appropriate interview](../../contributors/join-us/recruiti
 
 _Area Of Responsibility_ defines the area for which a [DRI](list-of-terms.md#dri) is accountable. The [AOR page](../../operations/operations/areas-of-responsibility.md) provides information on AORs across the company.
 
+### ARR
+
+Annual Recurring Revenue (“ARR”) is a measurement for time-limited subscription contracts. ARR normalizes contracted recurring revenue into a one year period to allow for like-kind comparisons between contracts that have differing start/end dates and/or different contract lengths. In a recurring revenue model, ARR represents the anticipated total revenue from all active contracts for the future 12 months from any measurement period. “Active” is defined as starting from customer contract signing date, not subscription start date. ARR does not include revenue from non-recurring items such as one-time setup fees or professional services.
```
Comment (jasonblais) [inline company/about-mattermost/list-of-terms.md:27]: Let's add this

```
For example, total ARR from 3 separate customers would be as follows:

Customer A: 6-month contract totaling $60,000. ARR = $120,000 ($60,000/6 * 12)
Customer B: 12-month contract totaling $240,000. ARR = $240,000
Customer C: 15-month contract totaling $45,000. ARR = $36,000 ($45,000/15 * 12)
Total ARR for Company = $396,000 ($120,000 + $240,000 + $36,000)
```

File: company/about-mattermost/list-of-terms.md [hunk 2]
```diff
@@ -24,6 +22,14 @@ Refers to both an [As Appropriate interview](../../contributors/join-us/recruiti
...
+
+### Bookings
+
+the ARR associated with sales transactions closed during the period. Note that Bookings and Billings are different, particularly for pro-rated contracts and multi-year contracts. Billings corresponds to the amount the customer is invoiced.
```
Comment (jasonblais) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
The ARR associated with sales transactions closed during the period. Note that Bookings and Billings are different, particularly for pro-rated contracts and multi-year contracts. Billings corresponds to the amount the customer is invoiced. 
```

Comment (jasonblais) [reply company/about-mattermost/list-of-terms.md]: Let's add

```
For example, Customer A signs a 3 year paid upfront transaction for $300,000. For Customer A, the bookings amount would be $100,000 ($300,000/3 years = ARR of $100,000) and the billings amount would be $300,000 (what we actually invoiced the customer at time of order signing). Customer B has a contract for 100 licenses at $50/year/license for 12 months, for total ARR of $5,000. They decide to add 25 more licenses after 4 months. For Customer B, the bookings amount would be $1,250 (25 * $50/year) and the billings amount would be $833.33 (25 * $50/year * 8 remaining months/12 months total contract).
```

**Review by jasonblais — COMMENTED** [review]
Just two comments, can you help with an update, else looks good, thanks Hanna!

**Comment by hannaparks** [comment]
Changes applied!

File: company/about-mattermost/list-of-terms.md (hunk 2 above)
Comment (jasonblais) [reply company/about-mattermost/list-of-terms.md]: @hannaparks did we apply this change? Sorry I realize there were two changes in one comment, which probably wasn't obvious

> Let's add

> For example, Customer A signs a 3 year paid upfront transaction for $300,000. For Customer A, the bookings amount would be $100,000 ($300,000/3 years = ARR of $100,000) and the billings amount would be $300,000 (what we actually invoiced the customer at time of order signing). Customer B has a contract for 100 licenses at $50/year/license for 12 months, for total ARR of $5,000. They decide to add 25 more licenses after 4 months. For Customer B, the bookings amount would be $1,250 (25 * $50/year) and the billings amount would be $833.33 (25 * $50/year * 8 remaining months/12 months total contract).

Comment (hannaparks) [reply company/about-mattermost/list-of-terms.md]: Yes we did :)

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
Thanks @hannaparks !

**Pull request merged**